sdk/nodejs/server/server.ts

```diff
@@ -0,0 +1,131 @@
+import ts from "typescript"
+
+import Client from "../api/client.gen.js"
+import { UnknownDaggerError } from "../common/errors/UnknownDaggerError.js"
+import { connect } from "../connect.js"
+import { entrypoinsMetadatatoGQLSchema } from "./convertor.js"
+import { EntrypointMetadata } from "./entrypointMetadata.js"
+import { serializeSignature, serializeSymbol } from "./serialization.js"
+import { listFiles, writeFile, readFile } from "./utils.js"
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type Entrypoint = (client: Client, ...args: string[]) => any
```

Comment:
should ...args be optional since you're not using with compute for instance ?
Here: https://github.com/dagger/dagger/pull/5305/files#diff-5ed95a013d71c623984de86737f366350b9a6a82e1b2e17e11b310da9cc5060bR9

Reply:
Nope, args is a destructured so it's actually optionnal because there can be 0, one or as many args as you want :)